release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -147,6 +147,12 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-nw-tls-profile"]
+==== Support added for TLS 1.3 and the Modern Profile
+
+This release adds Ingress Controller support for TLS 1.3 and the `Modern` profile.
```

Comment:
This release adds Ingress Controller support for using TLSv1.3 and the `Modern` profile in haproxy.